### DISCUSSION TITLE
Building Docker images through DockerFile is slow

### DISCUSSION THREAD

**Discussion by Lxx-c** [langgenius/dify#7030] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Building Docker images through DockerFile is very slow, sometimes even taking 2 hours. The most important thing is that the packaged Docker image can reach up to 2 GB, and the slowest point should be at the place where the dependency package is downloaded from Debian. Is there any way to solve this problem?

**Comment by crazywoola** [comment]
You can set up the docker registry.

**Reply by Lxx-c** [reply]
I use Alibaba's Docker image proxy service, which performs well in quickly pulling configuration images. However, during the image building process, when the apt-get command needs to be executed to download and install dependencies from the Debian official repository, the speed slows down significantly, causing the entire building process to be prolonged.
